csharp/src/Ice/TcpAcceptor.cs

```diff
@@ -51,10 +51,10 @@ internal TcpAcceptor(TcpEndpoint endpoint, ObjectAdapter adapter)
 
             _addr = Network.GetAddressForServerEndpoint(endpoint.Host,
                                                         endpoint.Port,
-                                                        endpoint.Communicator.IPVersion,
+                                                        Network.EnableBoth,
                                                         endpoint.Communicator.PreferIPv6);
 
-            _socket = Network.CreateServerSocket(false, _addr.AddressFamily, endpoint.Communicator.IPVersion);
+            _socket = Network.CreateServerSocket(false, _addr.AddressFamily, Network.EnableBoth);
```

Comment:
We can already remove the `ipVersion` param of `CreateServerSocket` and simplify the implementation it is only called with `Network.EnableBoth`